pkg/types/cose/cose_test.go

```diff
@@ -17,10 +17,13 @@ package cose
 
 import (
 	"context"
+	"crypto"
+	"crypto/x509"
 	"errors"
 	"testing"
 
 	"github.com/go-openapi/swag"
+	"golang.org/x/crypto/openpgp"
```

Comment:
might need to use the protonmail dep instead.

Reply:
No longer exported